src/PythonBindings/VectorPyList.hpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <vector>
+
+#include <boost/python.hpp>
+#include <boost/python/stl_iterator.hpp>
+#include <boost/python/suite/indexing/vector_indexing_suite.hpp>
+
+namespace bp = boost::python;
+/*!
+ * \ingroup PythonBindingsGroup
+ * \brief Convert a bp::list of bp::object& to a std::vector
+ */
+namespace py_bindings {
+template <typename T>
```

Comment:
Could you add a C++ test case that verifies that if you convert a `std::vector<int>` to a `bp::list` and back that you get what you started with? I don't think there's more to test for these functions than that :)

Reply:
Should there be a testing directory for `PythonBindings`, is it just that one doesn't exist because there are no tests to run currently for `PythonBindings`?

Reply:
During conference call discussion we decided to delay writing the tests until I'm back in Ithaca and can help in person.